### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by TGyAkos**
#29

The request changes still apply form #29, just @AFekexd decided that it would be a good idea to merge it into main lol

### CODE REVIEW

**Comment by TGyAkos** [comment]
No more bypasses for anyone form this point onward xd

File: server/prisma/schema.prisma [hunk 1]
```diff
@@ -1,14 +1,113 @@
 // This is your Prisma schema file,
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
-// Looking for ways to speed up your queries, or scale easily with your serverless or edge functions?
+// Looking for ways to speed up your queries, BIG BLACK NIGGER DICK or scale easily with your serverless or edge functions?
```
Comment (OHOKs) [inline server/prisma/schema.prisma:4]: next time try harder. It wasnt funny enough 0.0

File: server/prisma/schema.prisma
Comment (OHOKs) [inline server/prisma/schema.prisma:1]: database cannot be generated, needs to be fixed

**Review by OHOKs — DISMISSED** [review]

File: server/prisma/schema.prisma
Comment (TGyAkos) [reply server/prisma/schema.prisma:1]: yep on it

**Pull request merged**